components/sidebar/sidebar_channel/sidebar_channel.tsx

```diff
@@ -173,11 +180,12 @@ export default class SidebarChannel extends React.PureComponent<Props, State> {
                     index={channelIndex}
                 >
                     {(provided, snapshot) => {
+                        const {onTransitionEnd: draggableTransitionEnd, ...draggableProps} = provided.draggableProps;
```

Comment:
draggableProps also injects onTransitionEnd, so we need to extract it from the props and pass on in our function